js/main.js

```diff
@@ -1,5 +1,6 @@
 import { Player } from '/modules/player.js';
```

Comment:
We are missing some `dots` here.
`Error: modules not found.`
![image](https://user-images.githubusercontent.com/31897434/73710279-d3ac9b80-4703-11ea-9515-8af11076d2f6.png)

kindly message on slack if after 5-10 mins you couldn't get to fix it. 👍

Reply:
Hi Oluwaseun,

I think I have solved it.

Regards